src/pages/develop/promises.mdx

```diff
@@ -83,13 +107,20 @@ Here's an example that awaits a promise, then decodes the payload from JSON form
     const payload = JSON.parse(new TextDecoder().decode(byteArrayPayload))
     ```
     </Tabs.Tab>
+    <Tabs.Tab>
+    ```rust
+     use golem_rust::await_promise_json;
+
+     const payload: String = await_promise_json(&promise_id);
```

Comment:
`String` is a bit confusing example here - as the `await_promise_json` deserializes the payload but with using String as an example one may think that it returns the JSON string itself. Let's use a made up type name like `MyPayload` or something.

Reply:
Sure